pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -425,3 +425,22 @@ def test_index_map(self, name):
             ((2018,), range(1, 7)), names=[name, name]
         )
         tm.assert_index_equal(index, exp_index)
+
+    def test_split_non_utc(self):
+        indices = pd.date_range("2016-01-01 00:00:00+0200", freq="S", periods=10)
+        split_indices = np.split(indices, indices_or_sections=[])
+        valid_indices = DatetimeIndex(
+            [
+                "2016-01-01 00:00:00+02:00",
+                "2016-01-01 00:00:01+02:00",
+                "2016-01-01 00:00:02+02:00",
+                "2016-01-01 00:00:03+02:00",
+                "2016-01-01 00:00:04+02:00",
+                "2016-01-01 00:00:05+02:00",
+                "2016-01-01 00:00:06+02:00",
+                "2016-01-01 00:00:07+02:00",
+                "2016-01-01 00:00:08+02:00",
+                "2016-01-01 00:00:09+02:00",
+            ]
+        )
```

Comment:
this is the same as `indices` but without a a freq, right?  if so, can you construct it as

```
expected = index.copy()
expected._set_freq(None)
```

Reply:
Yes, thanks